src/components/AllServices/AllServicesBundle.tsx

```diff
@@ -35,19 +35,19 @@ const AllServicesBundle = ({ title, description, navItems }: AllServicesBundlePr
   const itemLearningResources = items.find((item) => item.title === 'Learning Resources');
   const itemDashboard = items.find((item) => item.title === 'Dashboard');
 
-  const findIcon = (title: string) => {
-    switch (title) {
-      case 'OpenShift':
+  const findIcon = (bundleId: string) => {
```

Comment:
So looking at this. We don't need this function to be running on each render. Especially if we consider the bundle Id will not change.

The node can be created directly instead of firing a function and memo the result should prevent unnecessary runs.
```js
const bundleIcon = useMemo(() => {
  switch (id) {
    case 'openshift':
        return <ServiceIcon icon={FavorableIcons.OpenShiftIcon} />;
     //...
  }
}, [id])


<SplitItem> {bundleIcon} </SplitItem>
```